src/main/java/org/jenkinsci/plugins/blueoceandisplayurl/BlueOceanDisplayURLImpl.java

```diff
@@ -80,17 +83,16 @@ public String getChangesURL(Run<?, ?> run) {
 
     @Override
     public String getJobURL(Job<?, ?> job) {
-        if (isSupported(job)) {
-            String jobPath;
-            if(job.getParent() instanceof MultiBranchProject) {
-                jobPath = Util.rawEncode(job.getParent().getFullName());
-            } else {
-                jobPath = Util.rawEncode(job.getFullName());
-            }
-            return getRoot() + "organizations/jenkins/" + jobPath + "/";
-        } else {
+        BlueOrganization organization = OrganizationFactory.getInstance().getContainingOrg(job);
+        if (organization == null || !isSupported(job)) {
             return DisplayURLProvider.getDefault().getJobURL(job);
         }
+        return getJobURL(organization, job);
+    }
+
+    private String getJobURL(BlueOrganization organization, Job<?, ?> job) {
+        String jobPath = job.getParent() instanceof MultiBranchProject ? job.getParent().getFullName() : job.getFullName();
```

Comment:
So this is where I think the problem is. Instead of ``getFullName`` it should be using:
```
Functions.getRelativeNameFrom(item, group);
```
where group is the `org.getGroup`

Like here:

https://github.com/jenkinsci/blueocean-plugin/blob/8fb7b65591aeca64512c7b6e022befcb20cd6aa6/blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractPipelineImpl.java#L177-L194